whelktool/scripts/cleanups/2019/05/delete-Jslu-from-UR-Access/delete-Jslu-from-UR-Access.groovy

```diff
@@ -0,0 +1,29 @@
+/*
+ *
+ * This script deletes holdings for Jslu based on file input
+ * containing UR-Access bib-ids.
+ *
+ * See LXL-2247 for more info.
+ *
+ */
+String SIGEL_TO_DELETE = 'Jslu'
+String BIB_ID_FILE = 'lxl-2247-ids.txt'
+
+PrintWriter failedHoldIDs = getReportWriter("failed-to-delete-holdIDs")
+PrintWriter scheduledForDeletion = getReportWriter("scheduled-for-deletion")
+File bibIDsFile = new File(scriptDir, BIB_ID_FILE)
+
+selectByIds( bibIDsFile.readLines() ) { bib ->
+    String query = "id in (select id from lddb " +
+            "where data#>>'{@graph,1,itemOf,@id}' = '${bib.graph[1][ID]}') AND " +
+            "collection = 'hold'"
```

Comment:
Just a thought (but no need to change): a multiline-string (`""" ... """`) for the query might be nice. :wink: